static/routes.config.php

```diff
@@ -131,7 +131,7 @@
 	'/hashtag'            => [Module\Hashtag::class,      [R::GET]],
 	'/home'               => [Module\Home::class,         [R::GET]],
 	'/help[/{doc:.+}]'    => [Module\Help::class,         [R::GET]],
-	'/ignored/{id}'       => [Module\Item\Ignored::class, [R::GET]],
+	'/ignored/{id}'       => [Module\Item\Ignore::class, [R::GET]],
```

Comment:
I expect you to change the module path as well 😅

Reply:
It was early in the morning and of course you're right ;-)